### PR TITLE
DR-1887 Add some logging and more transactions to get snapshot

### PR DESCRIPTION
**Pull request by nmalfroy**

### CODE REVIEW

File: src/main/java/bio/terra/app/controller/SnapshotsApiController.java [hunk 1]
```diff
@@ -183,7 +183,9 @@ private AuthenticatedUserRequest getAuthenticatedInfo() {
             defaultValue = RETRIEVE_INCLUDE_DEFAULT_VALUE
         ) List<SnapshotRequestAccessIncludeModel> include
     ) {
+        logger.info("Get auth");
```
Comment (pshapiro4broad) [inline src/main/java/bio/terra/app/controller/SnapshotsApiController.java]: Should these be `debug`? That way we could leave them in place after this has been debugged.

**Review by pshapiro4broad — APPROVED** [review]

File: src/main/java/bio/terra/app/controller/SnapshotsApiController.java (hunk 1 above)
Comment (nmalfroy) [reply src/main/java/bio/terra/app/controller/SnapshotsApiController.java]: I'm actually going to take these out.  Sorry...didn't know if putting in draft would make the tests not run.  Just wanted those in there to verify that the transaction annotation might work...looks like so far I'm 3/3 on it...

Comment (nmalfroy) [reply src/main/java/bio/terra/app/controller/SnapshotsApiController.java]: Once the connected test is done, I'll rebase and push without the log

**Review by samanehsan — APPROVED** [review]
Lgtm! ⛅

**Pull request merged**